sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerLiveTestBase.cs

```diff
@@ -46,9 +46,9 @@ protected async Task SleepIfNotInPlaybackModeAsync()
                 await Task.Delay(10000);
```

Comment:
There are few Tests in Record mode required human interaction(i.e. pick up the call to let the call in established state, so that further Api operations can be performed). 
Alternative ways is using debug break point.

Reply:
👍. You can renaming the method then to `WaitToPickUpPhoneInPlaybackModeAsync` to make it more clear